scipy/special/tests/test_support_alternative_backends.py

```diff
@@ -56,6 +56,14 @@ def test_rel_entr_generic(dtype):
                                     [(10,), (11, 1), (12, 1, 1), (13, 1, 1, 1)]])
 def test_support_alternative_backends(xp, f_name_n_args, dtype, shapes):
     f_name, n_args = f_name_n_args
+
+    if (SCIPY_DEVICE != 'cpu'
+        and is_torch(xp)
+        and f_name in {'stdtr', 'betaincc', 'betainc', 'chdtr'}
```

Comment:
aha, I knew something was up - `torch.special` has `gammainc`, so why can it not use the generic `_chdtr`? Answer:

https://github.com/scipy/scipy/blob/b841243c1f05e475e3098dee58cfb721b6912c2a/scipy/special/_support_alternative_backends.py#L149

I'll fix it here.